### PR TITLE
New package: melt-0.4.0

### PR DESCRIPTION
**Pull request by dmarto**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, **x86_64-glibc**

### CODE REVIEW

File: srcpkgs/melt/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'melt'
+pkgname=melt
+version=0.3.0
+revision=1
+create_wrksrc=yes
```
Comment (classabbyamp) [inline srcpkgs/melt/template:5]: the github-built release archive (`https://github.com/charmbracelet/melt/archive/refs/tags/v${version}.tar.gz`) has exactly the same contents, but doesn't need create_wrksrc

Comment (dmarto) [reply srcpkgs/melt/template:5]: indeed, but I prefer to use the one made and provided by the maintainers

**Comment by dmarto** [comment]
updated to new release

**Comment by dmarto** [comment]
With the release of Drew DeVault's Himitsu, I believe to have found a better workflow; thus, loosing interest in `melt`.

**Pull request closed without merging**